### DISCUSSION TITLE
如何让firefox 通过 wayland 使用fcitx5？

### DISCUSSION THREAD

**Discussion by guyuming76** [fcitx/fcitx5#523] [Q&A] [answered]
我可以在dwl上，firefox里，可以使用中文输入法搜索，但我发现firefox是通过dbus和fcitx5通讯的。
如果我在启动fcitx5时加入 --disable dbus 参数，就没法输入中文了，输入法选词框还会弹出，但选择的中文不会显示在firefox里。

这个是firefox本身的问题吗？还是我在gentoo上装firefox 时没选择正确的USE flag?

![20220614_18h08m22s_grim](https://user-images.githubusercontent.com/12045548/173479675-c0ba59b8-5cb5-45bd-8230-5c9ce9e64170.png)

**Comment by wengxt** [comment, marked as answer]
GTK_IM_MODULE=wayland
